### ISSUE TITLE
Request plexWatch Support

### ISSUE THREAD

**Issue by mostlydave** [styxit/HTPC-Manager#325]
Built in support for plexWatch would be awesome! plexWatch let's you see exactly what's being used by your friends plex is shared with:

https://github.com/ljunkie/plexWatch

**Comment by Hellowlol** [comment]
You might wanna include some info about what you info you want from plexwatch. Atleast i didnt get it. Its possible to see what your friends are watching in in the plex module already.

**Comment by Clock** [comment]
I would like to see the sorting of the library like XBMC and the direct link from the media-details to the Plex detail-page! (I don't think this needs PlexWatch to be honest)

**Comment by Hellowlol** [comment]
https://code.google.com/p/plex-api/wiki/PlexWebAPIOverview See advanced filtering

**Comment by tester22** [comment]
You can already see what users are playing and in the current dev branch it is possible to se what payer they are using. https://github.com/tester22/HTPC-Manager/commits/plex-queue

Adding plexwatch might be to much...

**Comment by Hellowlol** [comment]
@tester22 while your at it you should change how the getthumb works, atm it will always transcode images via plex. That reduces image quality.

**Comment by tester22** [comment]
@hellowlol yes. Like all other integrations the images displayed to the user are transcoded. I have been using the plex transcoder in favor of using the built in transcoder since it is much faster and no external dependencies. I don't think it would be better to transcode internally.

**Comment by Hellowlol** [comment]
I'd say is marginally faster, the biggest overhead is the http calls. While it's not visible on a low res screen the images has a lesser quality resulting in ugly images on high res screens. Also the opacity parameter is missing from the URL modal text is very hard to read. 

In my fork we check if pil is installed if not we fall back to transcode. I also fixed the play command to you can push videos to iPhones etc.

**Comment by tester22** [comment]
@hellowlol Will have a look and can push to your version. I have rewritten all play commands in my latest version to use the same approach as Plex Media Served does.

**Comment by Hellowlol** [comment]
The stuff about the images isn't merged yet but I plan to do that today and revert the stuff regarding dupes and make that a optional.
